Dockerfile.fedora-25-master-nightly

```diff
@@ -53,6 +53,7 @@ RUN systemctl enable ipa-server-configure-first.service
 COPY exit-via-chroot.conf /usr/lib/systemd/system/systemd-poweroff.service.d/
 
 RUN groupadd -g 288 kdcproxy ; useradd -u 288 -g 288 -c 'IPA KDC Proxy User' -d '/var/lib/kdcproxy' -s '/sbin/nologin' kdcproxy
+RUN groupadd -r ipaapi ; useradd -g ipaapi -c 'IPA Management Framework User' -d '/var/lib' -s '/sbin/nologin' ipaapi
```

Comment:
Please specify uid / gid.

You have to do it or next image might pick different numbers and when you then bind-mount volume which has data from previous image, file ownership will be wrong.

Unlike of on-host installations where you install once and then upgrade software and keep the uid/gid if there already is one, with container there are no upgrades happening.